src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,269 @@
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Focus
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-focus-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Selected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-selected-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Disabled
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-disabled-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+        </table>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      
+      <div class="sky-text-block">
+        Keep titles and descriptions short and concise.  Limit descriptions to a single sentence.
+      </div>
+      
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      
+      <sky-docs-design-guideline>
+        <p>
+          Selection boxes are arranged in a centered container based on their number and the container’s width. Don’t use different layouts with selection boxes.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do center selection boxes and match their heights."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-layout-1-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use custom flex layouts to arrange selection boxes."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-layout-2-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        <ul>
+          <li><a href="https://developer.blackbaud.com/skyux/components/checkbox">Checkbox</a></li>
+          <li><a href="https://developer.blackbaud.com/skyux/components/radio">Radio button</a></li>
```

Comment:
After looking again at the Development tab docs, I think we definitely want to include a link to the icon docs because users need to use it to include the icons.
```suggestion
          <li><a href="https://developer.blackbaud.com/skyux-indicators/docs/icon?docs-active-tab=design">Icon</a></li>
          <li><a href="https://developer.blackbaud.com/skyux/components/radio">Radio button</a></li>
```